modules/sr/robot/motor.py

```diff
@@ -25,11 +25,14 @@ def init_motor_array(webot: Robot, robot_type: RobotType) -> List[Motor]:
             ),
         ]
     else:
-        # TODO: placeholder crane motor board
         return [
             Motor(
-                Wheel(webot, 'left wheel'),
-                Wheel(webot, 'right wheel'),
+                LinearMotor(webot, "bridge motor"),
+                LinearMotor(webot, "trolley motor"),
```

Comment:
Question: Which is which? Are we defining this nicely somewhere for humans?

Reply:
I was using this diagram
![Gantry crane](https://images.squarespace-cdn.com/content/v1/5bde037836099b2eb8cf8e27/1542317754070-7TGVI2A6I6EXFMWVPZFQ/Overhead+Bridge+Crane%2C+Crane+Engineering%2C+Overhead+Crane+Conusltants+LLC?format=1500w)